Update lambda_function.py

ishitatyagi14 opened the pull request:
- Added write store_id & customer_id to dynamo db

ishitatyagi14 committed:
Update lambda_function.py

- Added write store_id & customer_id to dynamo db

rehan141196 commented on api/lambda_function.py:
```diff
@@ -11,8 +11,17 @@ def lambda_handler(event, context):
-
-    response = {}
+    # write store_id & customer_id to the DynamoDB table using the object we instantiated and save response in a variable
+    response = table.put_item(
```
I don't think we need this here because we don't want to insert into the DB every time the lambda is called. We want to handle based on the type of request we receive as we see in the if statement block below

ishitatyagi14 replied:
aah gotcha makes sense

rehan141196 commented on api/lambda_function.py:
```diff
@@ -11,8 +11,17 @@ def lambda_handler(event, context):
+            'customerId':customer_id
+            })
+    # return a properly formatted JSON object
+    return {
```
Having a return here will mean that nothing below this will ever execute which is not what we want

rehan141196 commented on api/lambda_function.py:
```diff
@@ -11,8 +11,17 @@ def lambda_handler(event, context):
+    # write store_id & customer_id to the DynamoDB table using the object we instantiated and save response in a variable
+    response = table.put_item(
+        Item={
+            'storeId': store_id
```
The database schema is StoreId, CustomerId, Stamps. So storeId should be capitalized to StoreId and so on

rehan141196 reviewed (changes requested):
Added some comments on the logic and syntax. I also don't think we need to change the lambda_handler function right now because I set it up with my previous PR #8. It would be better to pick one of the unimplemented functions and implement those

ishitatyagi14 committed:
Implemented changes from #9

Deleted the # write store_id & customer_id to the DynamoDB table using the object we instantiated and save response in a variable
